treaded_base_node:
  gains_package: "hebi_description"
  gains_file: "config/gains/r-tready-gains.xml"